src/main/java/cz/brmlab/yodaqa/flow/dashboard/QuestionDashboard.java

```diff
@@ -26,7 +26,7 @@ private QuestionDashboard() {}
 	public static QuestionDashboard getInstance() {
 		return qd;
 	}
-
+	private Map<Integer, AnsweringSnippet> snippets = new HashMap<>();
```

Comment:
So we have a global list of snippets, but per-question list of sources?

That's strange. What is the rationale? I think we should have both either per-question or global. (Not sure which, maybe per-question makes more sense for now?)